plugins/turing/runner/experiment_runner.go

```diff
@@ -37,8 +37,7 @@ func init() {
 
 // experimentRunner implements runner.ExperimentRunner
 type experimentRunner struct {
-	httpClient *xpclient.ClientWithResponses
```

Comment:
There is no need for this client since the experiment runner no longer contacts the central Treatment Service.